### DISCUSSION TITLE
Thinking Cap #5

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/22S-ASIAAM-191A#29] [Assignments]
### Reminder: Responses submitted after Sunday 2pm will not be eligible for awards.

> ## What do you think about design justice? How might you apply it in your designs going forward?

The reading on [Design Justice](https://albertkun.github.io/22S-ASIAAM-191A/materials/readings/design_justice.pdf) will be important to completing this thinking cap.

## Submission
Make a new comment below and respond to two students comment before class begins.

**Comment by daikis12** [comment]
I thought the reading about design justice is quite interesting and thought-provoking as I hadn’t thought of design as something that can reflect different forms of injustices and inequalities found in society. I found that one of the author’s main arguments, which was that professional designs were very hierarchical and thus mainly contain narrow white male, heteronormative, and middle/upper-class perspectives, convincing because it is true that many occupations related to design (e.g., software development and architecture) require a high level of education. Therefore, people from a low-income background and/or historically marginalized communities are often absent in these types of fields of employment. This inevitably leads to the production of design that lacks diversity in perspectives and experiences, and this further exacerbates the hierarchical structure in professional design. This is obviously problematic as “design mediates so much of our realities and has tremendous impact on our lives” (p. 2). Therefore, design justice is critical as its objectives attempt to deconstruct the current hierarchical nature of professional design and make it more accessible and inclusive. I particularly like that design justice “prioritize design’s impact on the community over the intentions of designers” (p. 6).  I believe this is an important approach to design as its emphasis is on the people and community, and not the creator him/herself. This is very similar to how we have been preparing our group project in class since our focus is on the community that we are surveying and not the solutions that we think are important (which generally contain our biases and egos). I can apply the principles of design justice in my design going forward by always prioritizing the inclusion of people and thinking about how my design can empower them.

**Reply by joonyloony** [reply]
I agree that it's important to prioritize the design's real life impacts over the intentions, as many times even though the intentions are innocent enough, it may stem from the designer not doing proper research or make the proper considerations before creating something that may do tremendous harm to marginalized communities.

**Comment by kericsp** [comment]
I love that the design justice reading includes excerpts from Kimberle Crenshaw's single-axis framework and the matrix of domination. I think that understanding the history of injustice against marginalized groups is a vital priority for any sort of design in software, from startups to digital institutions like Facebook. In particular, knowing that biases against race, class, and gender have been hard-coded into our justice system and governing institutions can help to humble us as software designers (a role with lots of inherent privilege due to educational status, which is itself tied to racial and class factors, as mentioned in the readings). This is essential in keeping the end-user top-of-mind; "The kids in Startup House may want a pizza delivery drone, but not in the same way low income families want health care, or the elderly men lying in their own faeces on Howard Street want a safe place to sleep" (p.9). This is particularly poignant when considering examples such as Facebook flagging "fake names"-- this technology is something that could very easily be co-opted by the government to perhaps throw out ballots of POC with non-European names, a threat that is very real in our current political climate. However, I think there is a fine line between listening to the end user and ensuring that efforts aren't wasted in creating a "solution" that doesn't actually solve anything, and harnessing the collective efforts of communities and coders to take steps towards creating actual solutions. This goes back to the idea of technological determinism in society, wherein the drive to place technology on a pedestal as a be-all, end-all solution blinds us to the realities of society.Going forward, I will try to apply design justice by always listening to the community and advocating for their stated needs, rather than acting out of my own interests as a creator. In this way, I can hopefully help turn the tide against prescriptive design and towards descriptive, grassroots-oriented design.

**Reply by kris10fu** [reply]
I agree that, when creating my designs going forward, it should be more important to be listening to the impacted community rather than my own interests or desires as a designer. I think this is particularly critical to keep in mind, especially because of the privileges I have. Incorporating more discussion and education about personal biases regarding race, class, and gender within design in companies and businesses could be helpful in ensuring greater design justice in the corporate world.

**Comment by willa-21** [comment]
I appreciate that design justice is both a field of theory and practice, and as such “rethinks design processes, centers people who are normally marginalized by design, and uses collaborative, creative practices to address the deepest challenges our communities face”. I think it is important that this chapter bridges theory to practice, with a focus on the theoretical concepts of intersectionality and the matrix of domination, and continues to highlight the work of practitioners who care about design at the intersection of ethics, equity, and systemic oppression. Moving forward, I will be sure to follow the suggested pillars of design justice, with a particular focus on involving the communities at hand. Since design is “problem-solving” at its core, I think these principles are far more relevant than one might suspect. Something that comes to mind is how a course or syllabus is designed, using design justice as a framework can help educators be intentional in accessibility for students.

**Reply by codingdianat** [reply]
I like how you mention design justice in education! There are many art and design institutions out there that lack a course that teach about design ethics/justice. This leads to designers who utilized their skills without regard for its implications. Looking at UCLA's Design Media Arts course listings, I am having a hard time finding classes that teaches design ethics or justice. Pretty disappointing.

**Reply by ntran27** [reply]
I appreciate that it emphasizes theory and practice too. I think that's a claim made a lot in education yet some of the things we read are impractical. The article was really clear and concise with defining what design justice is while giving more tangible examples of where it would apply. That in itself can make the principles more digestible and accessible for use by those who may have never heard it before

**Comment by antonyblin** [comment]
After having read this week’s article, I have a much better understanding of what design justice entails: previously, I would create designs and graphics with only accessibility in terms of disability in mind, as I had very little concept of what kinds of inequalities might exist beyond the physical in this subjective practice. I also think that it is important to understand design not only as something presented expressly for consumption by the eye; urban planning, technology, graphic design, infrastructure, and much more are all significant parts of everyday life that have been modified or utilized with exploitative and unfairly dominant patterns. From this point forward, I plan to apply design justice in my own work by doing as best I can to focus on the needs and statements of the exploited and to brainstorm and consider (prior to even drafting my works) who and what I may need to think about as I design.

**Reply by joonyloony** [reply]
I relate to you! At first when I thought of 'design' I really only though of 'design' with relation to art or aesthetic purposes, but now I understand that design really applies to almost everything in our lives (as you said, urban planning and infrastructure, but also laws and social norms). I think that the very act of identifying that the concept of 'design' reaches further than just art and graphic design allows us to reconceptualize what it means by 'design justice' and make efforts to create more equitable designs.

**Reply by kericsp** [reply]
I agree with both this post and the first response! Design isn't limited to digital or graphic design, it applies to the way we build and structure everything around us and the way we think about the things we interact with every day. The point you bring up about designing for disabilities is interesting to me, because I had a very similar mindset wherein I would think about designing a platform to accommodate for disabilities in a way that closely emulates the experience of a fully abled person. However, the design justice reading tells us to consider whether it is even correct to try and fit all user experiences within a certain mold; maybe sometimes it is better to reconfigure the platform itself to better serve the purposes of its users rather than designing "prescriptively".

**Reply by angelineeow** [reply]
I relate to not having much knowledge of what kinds of inequalities are in this particular practice. I also have a better understanding of design justice after reading this article. I believe it is important to bring out the voices of those who needed help and refrain from those who try to exploit the circumstances to utilize what designers designed. We all need to think twice or thrice when designing if we are helping those who needed help and not for our own or others' benefits. Great job!

**Comment by bittertang** [comment]
I believe that design justice is a necessary part of design, because oftentimes visuals and aesthetics can overshadow the actual experience for the audience they’re intended to serve. Before starting the process of designing, it’s important to know who the people in the community are and how whatever is being designed will impact them. One great example that the article brings up is the binary gender dropdown during account profile creation; being able to provide an option for non-binary individuals is integral to ensuring that they feel included and seen. Simply having that as an option can make a user feel acknowledged in their identity, and that in turn can facilitate trust with the application. Therefore, it’s crucial to implement culturally relevant and appropriate design, especially when the product or application is intended for minority communities, who may often times feel a lack of representation. In my own designs going forward, I will invest the time and effort early on in the process to get to know the community that I’m designing for and what their needs and goals are.

**Reply by codingdianat** [reply]
I really liked how you brought up the gender dropdown! I remember my partner feeling recognized and happy when instagram allowed pronouns in the bio. It's crazy that something that is so simple to implement is not prioritized by tech companies in their early days. This is most likely due to the lack of diversity in the field of tech.

**Reply by antonyblin** [reply]
You have a really great example in the presentation of gender as a binary! Being innovative in how we enable people to demonstrate gender identity can potentially be affirming and even stress-reducing in the completion of a form or other article, not to mention how beneficial it is to simply represent marginalized individuals who do not conform to the gender binary. I'm glad to see that you wish to invest time early in the design process to meet impacted communities' needs and desires, and you have a great jumping off point with your example.

**Comment by emmashanson** [comment]
“Overall, design justice builds on the foundational work in values in design (VID). VID emphasizes that designers make intentional choices about the affordances and aesthetics of objects and systems that they create.” This reading was informative in that it also presented clear data to the demographics in those who work in the software engineering industry, referencing that it is mostly cisgender white and asian men, and one of the lowest demographics in that industry would be black/latino women. Before reading this article, design justice was never something that I had even considered, nor was a topic that I knew about. However, this article taught me about the injustices that happen that aren’t ever talked about, and going into my designs forward, I think that I would be a bit more intentional about any designs I plan to make in the future. Designs, especially those in tech may not seem to have political nuances or undertones but learning about there is in fact an intersectionality is great since it reinforces the idea that there is always something to be learned.

**Reply by kris10fu** [reply]
I also did not really think about design justice before and am really glad to have learned more about the field and social movement through this week's reading. I agree that designs are not often thought of as having political undertones. However, it is clear that these intersecting systems of race, gender, and class permeate all aspects of society, whether we are aware of it or not. I definitely think that I will be more cognizant of the social impacts of the designs I create on the affected communities going forward as well.

**Reply by kericsp** [reply]
I agree that the reading really made me think about the intentions and biases built into software systems and designs! Especially given the information on demographics that the reading presented, it's really eye-opening to see how much representation matters when creating anything that has an impact on the world (whether intentional or not).

**Reply by angelineeow** [reply]
I really like that you mentioned design justice is something you never considered because I felt the same too. I have not known about this topic until I read this article. I also agree that even though we have never gone over this topic before, I am still thrilled to apply it in my design later. Good work!!

**Comment by joonyloony** [comment]
I think that the concept of design justice is a direct way to fight against systemic injustices. When we think about the term ‘design’, we often think about it in artistic or aesthetic terms rather than a method to enforce and strengthen existing inequalities. When talking about design justice, we are thinking about “how design (affordances, objects, systems, processes) simultaneously distributes both penalty and privileges to individuals based on their location within the matrix of domination, and to attend to the ways that this operates at various scales,” such as the personal, community, or institutional level (Constaza-Chock). So, while design justice in itself is just a concept, it is pretty easy to identify design flaws and methods that reinforce certain binaries and marginalizations (such as “a binary gender dropdown during account profile creation”) and I believe that we can internalize and apply design justice at a personal level (and if possible, even through the community level). This seems daunting at first, but what it really boils down to is identifying ways that your creations and projects may uphold certain systematic issues, and actively choosing to change your designs in order to support marginalized communities. For example, if you are a discord chat admin, you can choose to ban certain words and change the speech norms within that community. If you are making a community survey for the Asian American studies department check-in, you can actively include demographic options that are usually not found in order to be as inclusive as possible. Personally, I would keep this theory of design justice in mind going forward for any project in the future, as well as when I am interacting with others at the interpersonal level or creating community-level surveys and polls for projects or research. It is true that it is very difficult for us to enforce design justice at an institutional level, but I believe that as long as we, as individuals and communities, make efforts to strive for a more equitable and uplift and encourage marginalized voices to break into large design institutions, we can slowly implement the design justice on the institutional level.

**Reply by emmashanson** [reply]
Hi, I really like how in the latter half of your post, you chose to provide examples of actions that can be taken, using the Discord Server as an example as well as the Asian American check-in.  Often times, even when I personally take surveys that ask questions about demographics, not all options are found and it seems that there are East Asian counties are represented but sometimes, South or Southeast Asian nations, especially smaller ones or smaller populations are not represented.  Like you concluded, smaller actions like this in order to be actively inclusive hopefully can serve as a domino effect at a higher level.

**Comment by codingdianat** [comment]
As an aspiring user experience design, design justice is a concept that I find extremely important to this field that is not mentioned enough. When I first started learning ui/ux, there were endless resources on how to become a "good" designer that masters the tools of design. Unfortunately, I found close to zero resources that mentioned the word "design justice". In fact, this is the first time I have ever heard of this phrase and as someone who hopes to design for a living I find that to be extremely disheartening. I will continue to learn more about design justice, specifically for UI/UX design, and incorporate it to my practices as a UI/UX designer. Most importantly, I will do my best to educate fellow UI/UX designers about the importance of this concept in hopes that they also incorporate it into their practices. One important UI/UX practice is User Research.

 One process that occurs during UI/UX is user research. Usually a ui/ux designer or ux researcher interviews potential users to based their designs based on that. What I usually find is that, as Laurie Penny puts it," The wants and needs of young, healthy, middle-class people with connections and a reasonable amount of spare cash are overrepresented" in these user interviews. This usually results in the development of technology that disregards the needs of those from marginalized communities. Like the paper states, designers are not "evil"  and intentionally isolate marginalized groups but are the product of a system that allocates resources to " design problems that affect the wealthiest groups of people" because it is more profitable. Design justice seeks to ensure a more equitable distribution of design’s benefits and burdens; fair and meaningful participation in design decisions; and recognition of community based design traditions, knowledge, and practices. Designing apps/websites with design justice in mind will lead to technology that is more accessible to marginalized communities instead of catering to just wealthy cisgendered white men. Since digital literacy is so important to accessing many resources (communication, therapy, medicine, transportation etc) prioritizing equitable access to technology through design will also help make those resources more accessible. A personal example is my parents struggle with using social media apps such as Facebook. As immigrants who were forced to abandon their family and friends home due to a war, accessing social media is crucial to staying connected to their roots. Even though the app is in the Vietnamese language, it is not accessible to them and they often have to rely on me to help them navigate it. Who is asking people like my parents about the usability of the app? Who is conducting user interviews with those who do not speak English?  Who is concerned about their access to technology?

**Reply by bittertang** [reply]
As someone that's also an aspiring UX/UI Designer, this was also my first time learning what "design justice" is, and it's definitely concerning that it's not a widely known concept in our field. I related to your personal example of how your parents struggle with using social media, and it goes to show the lack of user research as well as the lack of effort that companies put into ensuring that their products can be used by communities whose first language isn't English.

**Reply by emmashanson** [reply]
Hi, I really liked the example that you used with social media, and how it is not accessible for everyone even on the basis of language. The demand for social media usage is there, as seen with the demonstrated interest of all social media platforms across the world but with the app being catered towards a specific group of people, there will be people who end up being left behind though apps, specifically social media apps are created with the purpose to promote being able to stay in touch with people across the globe. Regardless of the size, all apps are designs should take into account the inclusivity aspect of it.

**Reply by willa-21** [reply]
I'm glad to know that those of you who are going into design, UX/UI are now aware of design justice principles. It's comforting to know that there is hope for more equity and justice-centered work, outside of traditional social science fields. I hope the curriculum will continue to progress and perhaps you can bring these concepts into your other classes!

**Comment by angelineeow** [comment]
The article mentioned, "Design justice focuses on the ways that design reproduces, is reproduced by, and/or challenges the matrix of domination." Overall, I think that every designer should look through design justice lenses when they design objects or systems as it is important to include everyone. Without design justice, many issues such as the concept of intersectionality and the matrix of domination would occur. The article also raised an important concern where diversifying the software workforce will not automatically produce a more diverse default imagined user. In today's society, I would assume that the intention of diversifying the tech workforce would bring huge benefits for all the workforce but in the U.S., only straight white middle-class cisgender men, with educational privilege and high technological literacy, citizenship, native English speakers, and so on would default to the dominant social group. As someone who is part of the marginalized community, I began to be aware of this problem as I read the article and hope that the designers who attempt to address problems would center the voices of those who are directly impacted by the outcomes of the design process. With this consciousness, I would also prioritize design’s impact on the community that I focus on and hopefully will empower the communities.

**Reply by ananyakodali17** [reply]
I agree that designers should be looking through the lens of design justice when creating things. I think that intersectionality is something that is discussed far too little, even though it affects so many things, including design. I agree that we should always focus on empowering the communities we are focusing on with our designs!

**Reply by daikis12** [reply]
I agree that diversifying the people who work for technology and design companies should improve the process of their design production as it should create an environment that fosters the development of unique ideas from people with different classes and perspectives.  As the article explains, the way those companies are currently run by limited people with privileges is problematic as it contributes to making the production of design more exclusive and discriminatory.

**Comment by ashley-lu** [comment]
After reading this paper, it's interesting to see how similar design justice is to critical cartography in terms of being aware who you're creating for and what you're trying to represent in your creation. Not only that, but the designer's role is that of a facilitator similarly to the role of a cartographer, and both roles should focus on allowing the community to lead initiatives. Ultimately, design justice really reinforces the fact that society and technology are intertwined, and you cannot design for technology without considering the implications on people, specifically on marginalized groups and communities. 

Thinking back on my own personal goals, I've always expressed interest in learning design because I see how integral it is for connecting user to technology. Part of the reason why I became interested in digital humanities is because of its relation to design; UI/UX, for example, plays a role in linking digital formats with humanities. However, I've only ever thought about design in terms of visuals and accessibility and making sure the user would know how to use the technology. I didn't realize that all of those little design decisions could make such a large impact in terms of further reinforcing power imbalances. For example, I've never really used Facebook that much, but learning how its algorithms distinguish between real and fake users based on name biases made me realize that every design choice has consequences. It's not just about making your design sleek and easy to use; design is about giving power back to the community.

Moving forward, I think that in my quest to learn design, I can't think about it only in terms of what I previously knew. Even for something like user research, which is about hearing the voices of the users and addressing a shared problem between them, survey design and user stories have the ability to silence voices and take power away from the community (as evidenced by the numerous revisions we've all had to do for the memo and survey, which have definitely helped me learn how to center efforts more on the community being represented). I think design justice is a good framework for introducing more socially conscious design into things we create, and to echo my previous sentiment, I'd like to think that I would apply design justice to my designs similarly to how I would apply critical cartography methods to my maps. I want to be able to interact with the communities I'm representing and understand how I can be a facilitator for creating things for them. Even for something like a survey, which I would not have thought much of previously, I want to think about how I might be marginalizing people and focus on making sure their voices are heard.

**Reply by ananyakodali17** [reply]
I agree that design justice is similar to critical cartography! This all does really just relate back to the key point of knowing your audience and empowering them rather than just creating something for the sake of it. I like your interpretation of design choices, because it should always be about giving back to the community.

**Reply by willa-21** [reply]
I like how you connected design justice to critical cartography! Great connection. I agree it is important to use design justice principles even when it comes to designing and implementing surveys. I think the pandemic has made us (at least in educational spaces) more aware of accessibility, but even so, there's always room for improvement in remaining community-centered.

**Comment by ananyakodali17** [comment]
I think that design justice is something that is not often discussed or considered, but is very important when designing things. Similarly to map making, designing things comes with a set of inherent biases based on your life experience as well. The paper mentioned that most things being produced assume a target audience of the dominant social group of an area, and here and now that means cishet white men who speak English fluently and have had access to higher education. As such, it is important to consider what you are actually designing for, and make the focus of your design around being inclusive of everyone that may benefit from your design. I will use what I have learned about design justice from the reading to check my biases when creating things from now on, and really think about who is the target and what they can benefit from my designs. The concept of design justice is something that should be discussed more frequently and made more apparent, as it is clearly evident by statistics that the "target audience" of the socially dominant group in America so heavily comprises the top positions of software development corporations, and this imbalance is reflected in the tools they produce.

**Reply by bittertang** [reply]
I really liked how you drew parallels to mapmaking and how we have inherent biases that can impact these processes. What we've learned so far about design and mapmaking all go back to centering and empowering communities, and our role in facilitating this should solely be to help provide a platform for them.

**Reply by daikis12** [reply]
I agree with you that the concept of design justice should be discussed more in general as it tackles numerous societal problems and inequalities through the lens of design, which is something that people see all the day. As you wrote, teaching design justice to the target audience should help them reconsider the way in which design is currently produced in society and make it more inclusive.

**Reply by lindseymardona** [reply]
I definitely agree that design justice is something that isn't often discussed or considered. I really haven't heard of it before this class, and the concepts are something that make so much sense when explained and broken down as in the framework of the article, but they definitely aren't impactfully present without intentionality. This demonstrates to some level that it's really quite important to talk about more in order to strive for equity and inclusivity.

**Comment by kris10fu** [comment]
I enjoyed how the article delved deeply into the principles, nuances, and complexities of design justice as both a growing field and social movement. The discussion around intersectionality and the matrix of domination demonstrates how design cannot be separated from the intersecting systems of race, gender, and class, and how this is often ignored when designing systems and technologies. As mentioned in the reading, design often centers around ‘default’ imagined users with certain privileges and identities, such as being white or middle-class, that are often very exclusionary and ignore the needs of marginalized communities. I can see how this rings true in designs of new technologies and systems today as the people who are creating them often fall into the same demographics as these 'default' imagined users. This leads to designs that may look aesthetically pleasing but lack accessibility or only cater to dominant values. The examples of how biased designs discriminate against certain groups, like the LGBTQ or people of color, signify how important it is to spotlight the needs of these underrepresented communities. Rather than catering to the dominant group, ensuring that the ones who will be most impacted by a poor or exclusionary design can allow for greater inclusivity and justice. I agree that directly involving communities who are most impacted by whatever issue is being designed for is a critical ethical principle that can lead to greater transparency, collaboration, and innovation. Further, providing support and guidance for individuals from these communities who are interested in designing is another way to ensure greater design justice and equality within the field. 

I can apply design justice in my designs going forward by making sure to talk with the impacted communities and involve them in the entire process. I think that collaboration with such communities is integral and can allow for meaningful discussions and ideas on how to improve the designs. Also, seeking feedback from these communities about the effectiveness of the designs I create is another way to ensure that alternative values are encoded. This active process can also be a way to check my own privileges and biases when it comes to how I design. Another aspect of design justice that I can apply to my designs is focusing on all aspects of design and its larger social impact rather than merely on its aesthetics. I think that much of what I know about design, especially in terms of UI/UX, focuses on surface-level appearance. However, it is also imperative to be thinking of the bigger picture in terms of how to incorporate elements of design justice in order to create designs that empower rather than exclude.

**Reply by lindseymardona** [reply]
I like how you point out that aesthetic design is not everything. I think this is a mentality that I'm working on myself; aesthetics are something that are definitely a priority to me in terms of design, but it's important to acknowledge that aesthetic appeal cannot and should not stand on its own. Like you said, the context of larger impact should always be kept in mind when designing for a tool or servgice, and it should also be considered that a design should be usable, comprehensible, and accessible before aesthetic.

**Reply by stefuntran** [reply]
As someone who utilizes function over form ideology, I can attest to how design justice is looking to expand on the sentiment. Even as a website designer myself, I might not be making a space where anyone can feel comfortable because I did not know about the struggles from a certain community. On that same note that you mentioned, viewing our own work can be seen as a critique on the privilege we have and how to best give back to the community.

**Comment by lindseymardona** [comment]
I've never personally heard of the term "design justice" previous to this class, but to my basic understanding, design justice encapsulates questions such as "Who gets to do design?" and "Who are we designing for?" that aim to include marginalized groups in all stages of the design process, both in the sense of striving for workforce equity, as well as keeping in mind who the design is intended for use and to empower. I really like that the article defined design justice and then actually proposed a framework for design justice that included what is essential to keep in mind in order to make strides towards it. The part of the article that mentioned how it wasn't enough to diversify the workforce in order to diversify the 'default' imagined user for a product (which is a cishet white middle-class male with privileges such as moderately proficient tech literacy and speaking English at a native level) was really interesting to me; I feel that often times this idea of varying perspectives is one of the reasons that diversity in the workforce is always emphasized but there's less discourse about how this alone is not enough to solve the problem. In the design process specifically, there needs to be a level of intentionality involved.
In regard to my own designs going forward, I think in combination with the concept of the 10 Usability Heuristics, design justice supports this idea of accessibility and usability first. Before an aesthetic design, a website design should be easily comprehensible by the user, and this extends to varying levels of tech literacy. It's fine to consider aesthetic presentation a main priority, but there's no point in having an aesthetic design that isn't user-friendly to people who might not be as familiar with technology, and there's also no point in an aesthetic design for a tool that has no intentional purpose.

**Reply by NishuKrish** [reply]
I think you bring up an excellent point in that websites should be designed with people who may not have that much experience with technology. I feel like there is no use in creating an aesthetic website when it is unusable by community members. I think this something that we as students (including myself!) tend to do and that we should apply the principles of design justice to make our websites more fair and equitable.

**Reply by stefuntran** [reply]
I can agree with your experience on the term "design justice". I never thought about the making of a website in terms of the individuals or communities that make them. With this in mind, the design justice framework makes sense because we can see how reflective a website is based on who is apart of their team. The criteria of a reflective website is determined on their political views and the writing team in general.

**Comment by ntran27** [comment]
I found this week's article and design justice to be really interesting. I initially thought that design justice would be centered on the tech fields but I appreciate the examples throughout the article being connected to different objects and systems. As I was reading, I immediately thought about how design justice can be applied to the field of education. Specific to higher education, one of my research interests are in examining higher education's role and contributions to society. Many of the questions and points raised within design justice can be asked within higher ed. Who exactly was higher education designed for? Whose needs are prioritized? How does higher education replicate structures of oppression and inequities? Who is paid to lead the university and make the big decisions? Who is keeping these institutions and their million/billion dollar endowments accountable? Particularly as institutions strive to recruit more students from "diverse" backgrounds, who and what is keeping them accountable to the communities students are coming from? Recently, my former employer received a billion dollar donation to start a school of sustainability. It would be really interesting to apply design justice to answer some of the questions I posed and to analyze the development of this school.

**Reply by NishuKrish** [reply]
I definitely do agree that design justice can be applied to systems as a whole and not just websites/apps/etc. The major institutions in our life - from education to healthcare - have often been designed with those in power in mind and not the 99% of society. Hence, they seek to keep those in power in power and not keeping them accountable for what they are doing. I think you bring up a great point that design justice should be applied to how universities are trying to make their universities more diverse. Oftentimes, I feel like such diversity programs become overtly complex and fall short of their goal; they should be designed with the communities they are seeking to help in mind.

**Comment by NishuKrish** [comment]
I strongly do believe in the idea of design justice. This article did bring a number of points regarding how websites are not designed with marginalized communities in mind. Site designs are often made to be really confusing. Everything, from stylistic choices to the choice of vocabulary, can make a big difference in making a site more accessible or not. 
I have seen the impacts of some of the issues mentioned in the article regarding how there are a lot of design injustices. Last summer, I had the opportunity of working with a COVID-19 volunteer organization on vaccine hesitancy in Massachusetts. One of the big problems that was discovered through surveys with community members with the website we were using to spread information on vaccines was that the site itself was confusing due to the overtly technical jargon that was being used. This was leading to problems with our outreach efforts in various communities, especially poorer and rural communities where people may not have gone to college.
I feel like as students from a "prestigious" university, we sometimes do tend to get carried away when we do projects like these in order show off our educational background. We may make complex sites that may look good from an aesthetic perspective, but may not function that well. However, aesthetics are not the point of such projects. They are meant to empower the communities that are being harmed by the exploitative systems present in our world. I feel like this principle from the article encapsulates this: "We prioritize design’s impact on the community over the intentions of the designer". In any future projects I do, I will incorporate design justice by making sure community members have a voice in the making of the project. of the designer.

**Reply by penelopelamb** [reply]
Hi, I really liked how you included an example of where you have encountered design injustice. Your experience last summer is a great example of why design justice is important. In this case, the survey prioritizes technical jargon at the expense of marginalizing communities without higher education. This failure to incorporate design justice probably also impacted the reach of the survey and therefore its accuracy, which could have been avoided by ensuring that the communities it is intended for are considered and accommodated for at every stage in the process.

**Reply by XiongHer** [reply]
Thanks for sharing your experience, it really captures the importance of how design can better support the community, especially teaching communities of color about vaccines and covid-19 since there are so much misinformation out there. Often time people think that they did great things to the community without realizing the gap of understanding between the two parties, so it is important to listen and design a survey that best fit for the community. Aesthetic is important but functionality is key.

**Comment by stefuntran** [comment]
Coming from an ethnic studies background, the design justice framework intersects right into it. My experience of using the design justice framework was very limited before this class, and that demonstrates how new this concept is and limited it is taught in various technical classes. In order to make the internet a safe,accessible, and inclusive area for the marginalized community, it is important to learn about the different accommodations we need to implement. Learning these adjustments includes being proactive about it, and not just learning about it through feedback or requests. My future website should be based on functionality over aesthetic which includes the marginalized groups.

**Reply by penelopelamb** [reply]
I also had a limited understanding of design justice prior to this class. I like that you pointed out that it probably means that even though this class is full of designers or those with the potential to design, this concept is underemphasized. I think that awareness is the first step and it will push others to also prioritize functionality over aesthetic moving forward.

**Reply by ashley-lu** [reply]
I like how you connected design justice with your ethnic studies background. Frameworks that focus on inclusion of different communities are seldom seen in technical classes and I agree that we should be proactive in implementing conscious choices related to these communities when designing for them, as opposed to only considering aesthetics.

**Comment by ntran27** [comment]
I'm guilty of emphasizing aesthetics in the design of my work as well. Your post made me think of how it can be easy overlook or minimize all the ways a product or system needs to be flexibility and adaptable to all users. This is particularly true if we don't share the same background or experiences as we might not fail to consider how others might receive or use our work. I think this reality underscores the importance of collaboration and value of having many different perspectives provide feedback into our work in order to achieve design justice.

**Reply by spolcharoen** [reply]
I agree that it can be very easy to get lost in aesthetics and push usability/functionality to the side. I also like how you mention communication and collaboration as a way to counteract these design mistakes. UI/UX is all about usability testing. Fully testing your product with users is very important to making sure you meet their needs!

**Reply by kaittli** [reply]
In the beginning with design, it is easy to get caught up with the aesthetics because we are mostly taught just what is visually appealing. However, good design is not just about that but should also include its effects on the people who are consumers of the design as well as broader society. I think lacking exposure to certain communities can also fuel inequity in design, which is why it is important to get feedback on design by actually talking to marginalized communities. Including more voices and perspectives into design will promote design justice.

**Comment by penelopelamb** [comment]
I'd say that prior to this class, I had never truly considered design justice and its importance. At first I was under the impression that it would pertain mostly to ui/ux. I thought back to when Albert gave that presentation about how not prioritizing accessibility to change languages is harmful. Or, how google forms is not as compatible with non-English languages as say, its Microsoft counterpart. It was thought provoking to consider that a company overlooking these issues translates directly to what they are and aren't prioritizing. But, design justice extends beyond websites and ui/ux. I now understand that design justice encapsulates to every kind of creator and creation, and is an especially important for me to consider as an engineer. I think it is easy for creators to loose sight of the fact that if we are creating something targeted for a specific group and a specific purpose, we should also consider who it doesn't help and why.

**Reply by ashley-lu** [reply]
I love how you brought up examples from Albert's lecture; looking back on them, I realize that I'd never even thought about how those design choices could be harmful. Creators should definitely consider these implications when designing for a specific group and purpose in mind.

**Reply by XiongHer** [reply]
I agreed with your points and the connection you made to the class examples, I haven't really think of that. We often didn't recognize these small and subtle things on websites that have great impacts on society, and at the same time we take for granted since we can read and understand while others may not. So yeah, it is important moving forward to be mindful and critical of how we can better design more inclusive platforms.

**Comment by XiongHer** [comment]
I think design justice is an important approach moving forward to ensure the ethnics of technologies that we’ve been discussing/learning in these past few weeks. After reading the article, I realized that designers hold great influence and power to make significant impacts/change on society. The “Design Justice Network Principles” lays out some pretty good starting points, recognizing humanity in technologies, listening to marginalized people’s voices and concerns to heal and empower the community, among others. For the class project, my group plans to portray UCLA graduate students’ voices and experiences of their journey at UCLA and beyond. Our project will focus on the graduate education community and how students navigate higher ed rather than providing resources or solutions to a problem. This tool will empower other graduate students to share their experiences to support each other and to build their sense of belonging at UCLA and LA.

**Reply by antonyblin** [reply]
I like that you tied this week's reading and thinking cap to your actual upcoming project rather than your future works in general! I agree that designers are really very influential in our society, whether they are aware of it or not. That just goes to show how important the ethics we've been learning about in class are!

**Reply by spolcharoen** [reply]
I like how you related the reading and thinking cap topic of design justice to the class project! As shown in lecture as well, it is always important to take a step back and evaluate what community is being empowered by your work and if your project is reflective of your ethics. I am excited to see what your group comes up with!

**Comment by spolcharoen** [comment]
Being in the Digital Humanities minor has definitely made me more aware of design justice and how questions of ethics must be addressed when conducting projects. The reading also clarified this idea more clearly, bringing in a framework for design justice and showing how to implement it. Technology and society are interconnected, and thus, inequality and privilege exist within. Focusing on design justice through uplifting marginalized voices in tech and centering community empowerment is essential to creating equity in the tech space. It was interesting to read about the workplace example as well, as to me it shows the difference between striving for equality versus equity. I love how design justice takes design a step further from just UI/UX and considers what message is being conveyed and how it relates to the users. Going forward, I want to combine my prior knowledge in survey research and UI/UX design with the ethics lessons taught by Prof Albert to approach design projects with a more holistic lens/goal. As a designer, every choice can help make a user's life easier or harder, so I want to be more aware of the decisions I make and emphasize functionality in addition to aesthetics.

**Reply by kaittli** [reply]
I agree with your points how even though technology and society and so interconnected, there is still disconnect between technology and certain communities, but design justice can incorporate ideologies to help support the communities who are most in need. Design justice is the piece that is missing from many design processes and can help the technological side of design create a more positive impact on society.

**Comment by kaittli** [comment]
With design, many individuals may overemphasize the aesthetics or how cool the design is, but forget to think about how those kinds of designs may not be the most accessible to everyone. Design justice allows us to be mindful of the design that we are creating and encourages us to think about how it influences inclusivity and equity. The article contains an idea that I never really considered much before that stuck with me, which is that “design can distribute benefits and harms across members of society”. From a standpoint of visual design, applying design justice could entail using colors with high contrast or using less red/green colors for people with visual impairments. There is also design in the structures of the world with buildings and architecture that also can assist or hinder individuals. Design justice is important because it focuses on empowering different communities, or the people they represent, rather than only the designers themselves. As for my future with design, I will certainly be conscious of how design can impact communities, especially those that are marginalized, and to think of what my design is prioritizing. I think that design justice allows me to think more comprehensively about designs, not just looking on the surface level how “pretty” something is but rather the values and ethics employed through the design.
